lib/pg_saurus/connection_adapters/abstract_adapter/schema_methods.rb

```diff
@@ -5,12 +5,13 @@ module PgSaurus::ConnectionAdapters::AbstractAdapter::SchemaMethods
   # Provide :schema option to +create_table+ method.
   def create_table(table_name, options = {}, &block)
     table_name, options = extract_table_options(table_name, options)
-    super(table_name, options, &block)
+    super(table_name, **options, &block)
```

Comment:
Corresponding method in:
* Rails 6.0.0: [`ActiveRecord::ConnectionAdapters::SchemaStatements#create_table`](https://github.com/rails/rails/blob/v6.0.0/activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb#L294)
* Rails 6.1.0: [`ActiveRecord::ConnectionAdapters::SchemaStatements#create_table`](https://github.com/rails/rails/blob/v6.1.0/activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb#L296)

See that both take keyword arguments in their method signatures and **not** a positional hash. However, our method definition with a positional hash is fine (see the demo in the PR description for why).